customer-commons/src/main/java/no/unit/nva/customer/validator/ChannelClaimValidator.java

```diff
@@ -25,9 +30,17 @@ public static void validate(ChannelClaimDto channelClaim) throws BadRequestExcep
         if (isNotPublicationChannel(channelClaim.channel())) {
             throw new BadRequestException(INVALID_CHANNEL_MESSAGE);
         }
+        if (hasTrailingYear(channelClaim.channel())) {
+            throw new BadRequestException(TRAILING_YEAR_MESSAGE);
+        }
     }
 
     private static boolean isNotPublicationChannel(URI channel) {
         return !HOST.equals(channel.getHost()) || !channel.toString().contains(PUBLICATION_CHANNEL_PATH);
     }
+
+    private static boolean hasTrailingYear(URI channel) {
+        var lastPathElement = UriWrapper.fromUri(channel).getLastPathElement();
+        return nonNull(lastPathElement) && lastPathElement.matches(REGEX_FOUR_DIGITS);
+    }
```

Comment:
Might be tempted to just say "if we have a well-formed channel, anything else is not allowed" as atm "bla/blah" would be OK.